doc/api/fs.markdown

```diff
@@ -350,6 +350,10 @@ created. On POSIX systems, `path` is considered to exist even if it is a symlink
 to a non-existent file. The exclusive flag may or may not work with network file
 systems.
 
+`flags` can also be a number as documented by open(2); commonly used constants
```

Comment:
Not all users know what `open(2)` means. Would be nice if that was linkified or even better the numbers being documented here too.